programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -1040,7 +1069,12 @@ declare_builtin_function!(
         result_point_addr: u64,
         memory_mapping: &mut MemoryMapping,
     ) -> Result<u64, Error> {
-        use solana_curve25519::{curve_syscall_traits::*, edwards, ristretto, scalar};
+        use solana_curve25519::{
+            curve_syscall_traits::*,
+            edwards::{self, PodEdwardsPoint},
+            ristretto::{self, PodRistrettoPoint},
```

Comment:
why were these necessary? they add unnecessary noise to the diff

Reply:
Because it is a bit tricky to get a macro to correctly accept types and paths to types and still be able to peel off brackets around types. Thus, it was easier to not support paths and use `use` instead.